### PR TITLE
Release preps for 0.16.0

### PR DESCRIPTION
**Pull request by rhuss**

### CODE REVIEW

**Comment by dsimansk** [comment]
Thanks for polishing and double checking with merged PRs! :)

/lgtm

File: CHANGELOG.adoc [hunk 1]
```diff
@@ -12,20 +12,44 @@
...
+| 🎁
+| Add `--filename` flag to `service create` command
+| https://github.com/knative/client/pull/913[#913]
+
+| ✨
+| Find sink and ready conditions for untyped sources
+| https://github.com/knative/client/pull/911[#911]
+
+| 🐛
+| Add mandatory ``--sink` to PingSource command's name
```
Comment (navidshaikh) [inline CHANGELOG.adoc:50]:
```suggestion
| Add mandatory `--sink` to PingSource command's usage
```

File: CHANGELOG.adoc [hunk 2]
```diff
@@ -34,17 +58,21 @@
...
 
 | 🎁
-| Add commands to manage broker resource
-| https://github.com/knative/client/pull/894[#894]
-
-| 🐣
-| Refactor port field to accept port name and port
-| https://github.com/knative/client/pull/915[#915]
+| Add `kn export` type `Export`
+| https://github.com/knative/client/pull/866[#856]
```
Comment (navidshaikh) [inline CHANGELOG.adoc:75]:
```suggestion
| https://github.com/knative/client/pull/856[#856]
```

**Comment by rhuss** [comment]
@navidshaikh @maximilien updated, ptal.

**Pull request merged**